controllers/order.controller.js

```diff
@@ -0,0 +1,15 @@
+const User = require("../models/user.model");
+const jwt = require('jsonwebtoken');
+
+exports.fetchAllOrders = async (req, res) => {
+    const userId = req.body.user.userId;
+    try{
+        const result = await User.find({userId: userId}).populate('orders');
```

Comment:
Order controller should be returning orders of type order schema. Currently we are returning orders property from User schema

Reply:
Orders.find() operation as well which returns final list of orders associated with user